zerver/lib/actions.py

```diff
@@ -3562,6 +3563,20 @@ def get_non_subscribed_subs() -> List[Tuple[UserProfile, Stream]]:
             subs_to_deactivate.append(sub_info)
             sub_ids_to_deactivate.append(sub_info.sub.id)
 
+    stream_ids = [stream_id for stream_id in stream_dict.keys()]
+    stream_admin_dict = get_stream_admin_dict_for_streams(stream_ids)
+    stream_admin_count_dict: Dict[int, int] = {}
+    for stream_id in stream_admin_dict.keys():
+        stream_admin_count_dict[stream_id] = len(stream_admin_dict[stream_id])
+
+    for sub in subs_to_deactivate:
+        user_profile_id = sub.user.id
+        stream_id = sub.stream.id
```

Comment:
This should be `for sub_info in ...` ; we just `sub ` as a local variable only for Subscription objects, not SubInfo objects.